README.rst

```diff
@@ -187,6 +194,7 @@ Create database:
 .. code-block:: shell
 
     cd tests/
+    export WIFI_MESH=1
```

Comment:
is this always needed? Let's clarify if it's required, optional and what it does.

Reply:
I have updated the test project to avoid bloating the instructions